src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -13867,6 +13867,292 @@ public void Event_InReadOnlyStruct_ReadOnly_Add1()
                 SemanticEdit(SemanticEditKind.Update, c => c.GetMember<INamedTypeSymbol>("S").GetMember<IEventSymbol>("E").RemoveMethod));
         }
 
+        [Fact]
+        public void EventField_AddAttribute()
```

Comment:
>EventField_AddAttribute [](http://example.com/codeflow?start=20&length=23)

I'd reverse the naming and call this "NotSupportedByRuntime". To have consistent naming for tests of features that do not have specific runtime capability.

Reply:
For exampl,e here you'd write a combined test like so:

```
SemanticEdit(
  SemanticEditKind.Update,
   c => c.GetMember<IEventSymbol>("C.F"),
  capability: EditAndContinueTestHelpers.Net6RuntimeCapabilities,
  unsupported: Diagnostic(RudeEditKind.ChangingAttributesNotSupportedByRuntime, "event Action F", FeaturesResources.event_)
```

Reply:
Interesting idea. I think it would have to go on the VerifySemantics call though, as there are cases like delegate parameters where multiple edits are reported, and the alternative is a single diagnostic.

Reply:
Actually, we could just bring the `edits.VerifySemantics` call into this test, after `edit.VerifyRudeDiagnostics`, and it would have the same effect